stackdriver-errors.js

```diff
@@ -52,7 +52,10 @@ StackdriverErrorReporter.prototype.start = function(config) {
   this.projectId = config.projectId;
   this.targetUrl = config.targetUrl;
   this.context = config.context || {};
-  this.serviceContext = {service: config.service || 'web'};
+  this.serviceContext = {service: config.service || 'web',
+                         resourceType: config.resourceType || 'reported_errors',
```

Comment:
You are also adding resourceType in context. 
I am supportive, but I recommend sending a different PR for this.